src/main/java/com/juniorstart/juniorstart/payload/DeleteGoalRequest.java

```diff
@@ -0,0 +1,35 @@
+package com.juniorstart.juniorstart.payload;
+
+import com.juniorstart.juniorstart.payload.interfaces.InterfaceChangeRequest;
+import jakarta.validation.constraints.NotBlank;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import java.util.UUID;
+
+/** Class for delete user goal 26.09.2020
+ * @author Noboseki
+ * @version 1.0
+ * @since 1.0
+ */
+@Getter
+@NoArgsConstructor
+public class DeleteGoalRequest implements InterfaceChangeRequest {
+    @NotBlank
+    private UUID goalId;
+    @NotBlank
+    private UUID userPrivateId;
```

Comment:
Użytkownik nie ma pojęcia jakie ma UUID. Wie za to jakie ma Long userId. Tak samo powinno być z Goal. Czyli entity które zwracane są na front mają UUID jako domyślne ID i Long jako NaturalID. Czyli np. User ma te privateId i UserId i tak samo tu Goal powinien mieć privateId (albo Id nazwane, idk jak tam się stosuje, a dodatkowo Long goalId, które będzie zwracane na front)